Common/include/omp_structure.hpp

```diff
@@ -116,6 +116,7 @@ inline size_t computeStaticChunkSize(size_t totalWork,
                                      size_t numThreads,
                                      size_t maxChunkSize)
 {
+  if(!totalWork) return maxChunkSize;
```

Comment:
This fixes the issue if the code is compiled without OpenMP, which I guess for now is ok.
Do you still get the same information as before by not overwriting the history files? If so LGTM

Reply:
As the other stuff goes in with your PR, I think that should be Ok for most people. 
There was actually a small difference in the `--- Output Information ---` tab which is also shown in a normal run, but that is addressed now as well 👍